diff -up mingw-w64-v7.0.0/mingw-w64-tools/widl/include/basetsd.h.omv~ mingw-w64-v7.0.0/mingw-w64-tools/widl/include/basetsd.h
--- mingw-w64-v7.0.0/mingw-w64-tools/widl/include/basetsd.h.omv~	2020-02-16 17:50:27.321421442 +0100
+++ mingw-w64-v7.0.0/mingw-w64-tools/widl/include/basetsd.h	2020-02-16 17:51:57.277425408 +0100
@@ -309,6 +309,10 @@ typedef ULONG_PTR KAFFINITY, *PKAFFINITY
 # undef  WORDS_BIGENDIAN
 # undef  BITFIELDS_BIGENDIAN
 # undef  ALLOW_UNALIGNED_ACCESS
+#elif defined(__riscv)
+# undef  WORDS_BIGENDIAN
+# undef  BITFIELDS_BIGENDIAN
+# undef  ALLOW_UNALIGNED_ACCESS
 #elif !defined(RC_INVOKED) && !defined(__WIDL__) && !defined(__midl)
 # error Unknown CPU architecture!
 #endif
