diff -up mingw-w64-v7.0.0/mingw-w64-tools/widl/include/windef.h.omv~ mingw-w64-v7.0.0/mingw-w64-tools/widl/include/windef.h
--- mingw-w64-v7.0.0/mingw-w64-tools/widl/include/windef.h.omv~	2020-02-16 15:44:21.652087834 +0100
+++ mingw-w64-v7.0.0/mingw-w64-tools/widl/include/windef.h	2020-02-16 15:44:31.668088276 +0100
@@ -111,7 +111,7 @@ extern "C" {
 #endif
 
 #ifndef __ms_va_list
-# if (defined(__x86_64__) || defined(__aarch64__)) && defined (__GNUC__)
+# if defined(__x86_64__) && defined (__GNUC__)
 #  define __ms_va_list __builtin_ms_va_list
 #  define __ms_va_start(list,arg) __builtin_ms_va_start(list,arg)
 #  define __ms_va_end(list) __builtin_ms_va_end(list)
diff -up mingw-w64-v7.0.0/mingw-w64-tools/widl/src/widltypes.h.omv~ mingw-w64-v7.0.0/mingw-w64-tools/widl/src/widltypes.h
